testassets/ReverseProxy.Code/ReverseProxy.Code.csproj

```
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFrameworks>net5.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net6.0;netcoreapp3.1</TargetFrameworks>
```

Keep the 5.0 target in the samples.
```suggestion
    <TargetFrameworks>net6.0;net5.0;netcoreapp3.1</TargetFrameworks>
```